### PR TITLE
NEO-2262 Select chevron positioning and icon when expanded

### PR DESCRIPTION
**Pull request by yangAtSpoken**
[Link Select](https://deploy-preview-441--neo-react-library-storybook.netlify.app/?path=/story/components-select--basic-selects)

**Before tagging the team for a review, I have done the following:**

- [x] run `yarn all` locally: ensures that all tests pass, formatting is done, types pass, and builds pass
- [x] reviewed my code changes
- [x] updated the link at the top of this PR (or remove it if not applicable)
- [x] added any important information to this PR (description, images, GIFs, ect.)
- [ ] done an accessibility check on my work (tested with Chrome's `axe Dev Tools`, Mac's VoiceOver, etc.)
- [x] tagged `@avaya-dux/dux-design` if any visual changes have occurred
- [x] tagged `@avaya-dux/dux-devs`

`@avaya-dux/dux-design`: ADD DESIGN SPECIFIC BRIEF DESCRIPTION

- set chevron right margin to 12px per design;
- chevon icon toggles between down and up;
- fixed text jumping in a Select component is focused;

`@avaya-dux/dux-devs`: 

- css update

## Summary by CodeRabbit

- **New Features**
	- Improved UI for multi-select and single-select components to clearly indicate when menus are expanded.

- **Style**
	- Updated styling for inputs and multi-select headers to improve visual consistency.

- **Chores**
	- Upgraded `@avaya/neo` package in `devDependencies` from version `3.81.8` to `3.81.9`.

### CODE REVIEW

File: src/components/Select/Select_shim.css [hunk 1]
```diff
@@ -85,7 +85,8 @@ div.neo-multiselect:not(.neo-multiselect--small)
   span.neo-multiselect__padded-container
   div
   input.neo-input {
-  height: 36px;
+  height: 34px;
+  min-height: 34px;
```
Comment (joe-s-avaya) [inline src/components/Select/Select_shim.css:89]: it's a small red flag that you just updated the css library and then also need to update the shim file, but I won't block on it

**Review by joe-s-avaya — APPROVED** [review]
one comment but looks good! super happy to see a small and straight forward PR

**Review by enrique-prado — APPROVED** [review]

**Comment by Justin-Avaya** [comment]
It appears the Chevron is slightly off from the Figma design. The Figma design file uses an icon with an actual size of 16 x 16px, however the optical size of the icon (the chevron) is smaller and is nested inside the 16 x 16px icon. What appears to be displayed in the Storybook example given is the chevron icons used are 14 x 20px

See both screenshots and the quicktime .mov attachments below for a visual explanation.

Screenshot 2024-07-03 at 11 11 54 AM

Screenshot 2024-07-03 at 11 07 23 AM

https://github.com/avaya-dux/neo-react-library/assets/161497909/1699632e-ce11-461b-afe4-49b147e35727

**Review by matthewAvaya — APPROVED** [review]

**Review by Justin-Avaya — APPROVED** [review]
- [x] set chevron right margin to 12px per design;
- [x] chevon icon toggles between down and up;
- [x] fixed text jumping in a Select component is focused;

✅ Approved

**Comment by yangAtSpoken** [comment]
@Justin-Avaya Take another look, please.

**Pull request merged**